lib/src/Base/Common/ResourceMap.cxx

```diff
@@ -1584,6 +1584,12 @@ void ResourceMap::loadDefaultConfiguration()
 
   // viewer.View parameters //
   addAsString("View-ImageFormat", "png");
+
+  // SmolyakExperiment parameters //
+  addAsScalar("SmolyakExperiment-DefaultPointRelativeEpsilon", 1.0e-8);
+  addAsScalar("SmolyakExperiment-DefaultPointAbsoluteEpsilon", 1.0e-8);
+  addAsBool("SmolyakExperiment-MergeQuadrature", true);
+
```

Comment:
these should be added in openturns.conf.in as well

Reply:
This seems like a duplicated code, isn'it? Anyway, fixed.